examples/Examples.hs

```diff
@@ -19,18 +19,22 @@ import Streamly.Internal.Data.Unfold (Unfold)
 import qualified Streamly.Prelude as S
 import qualified Streamly.Data.Fold as FL
 import qualified Streamly.Data.Array.Storable.Foreign as A
+import qualified Streamly.FileSystem.Handle as FH
+import qualified Streamly.Unicode.Stream as U
 import qualified Streamly.Network.Socket as SK
 import qualified Streamly.Network.Inet.TCP as TCP
 
 import qualified Streamly.Internal.Data.Stream.IsStream as S
-import qualified Streamly.Internal.Unicode.Stream as U
-import qualified Streamly.Internal.Data.Fold as FL
+       (intercalateSuffix)
+import qualified Streamly.Internal.Unicode.Stream as U (words, unwords)
```

Comment:
We can replace these with their definitions and plan to expose `Stream.interpose` in 0.8.0.